sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.controller.getCompletedApplicationDetails.e2e-spec.ts

```diff
@@ -65,6 +75,52 @@ describe("ApplicationStudentsController(e2e)-getCompletedApplicationDetails", ()
       FakeStudentUsersTypes.FakeStudentUserType1,
       dataSource,
     );
+    student.sinValidation = createFakeSINValidation(
```

Comment:
The student used for authentication is shared and should not be changed.
When the student requires modification we should use the `mockUserLoginInfo`.